src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,78 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    private StringBuilder dataToWrite = new StringBuilder();
 
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> dataFromFile = readDataFromFile(fromFileName);
+        String finalArrayOfData = createReport(dataFromFile, new String[2],0,0);
+
+        writeDataToFile(toFileName, finalArrayOfData);
+    }
+
+    public List<String> readDataFromFile(String fromFileName) {
+        List<String> res = new ArrayList<>();
+        String line;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            while ((line = reader.readLine()) != null) {
+                res.addAll(Arrays.asList(line.split(",")));
```

Comment:
why do you need to split here? add whole line

Reply:
To write with commas